### PR TITLE
Add new `connectrpc/query-es` plugin

### PR DESCRIPTION
**Pull request by smaye81**
This adds the @connectrpc/protoc-gen-connect-query plugin to the plugins repo.

Note this can't be merged until the package is officially published on npm.

### CODE REVIEW

**Comment by smaye81** [comment]
@timostamm I named this `connectrpc/query-es` because I feel like `connectrpc/query` is a bit too vague and has a potential for name clashes down the road.

**Review by timostamm — COMMENTED** [review]
> `connectrpc/query-es`

Agree with the name, considering other names are `go` and `es`.

**Comment by timostamm** [comment]
Once packages are available in the `@connectrpc` scope, the lock file needs to be generated.

File: plugins/connectrpc/query-es/v0.4.2/package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "plugins-connectrpc-query-es",
+  "version": "1.0.0",
+  "dependencies": {
+    "@connectrpc/protoc-gen-connect-query": "0.4.2"
+  }
```
Comment (pkwarren) [inline plugins/connectrpc/query-es/v0.4.2/package.json:6]: We should also check in a package-lock.json file alongside this.

**Review by pkwarren — APPROVED** [review]

**Pull request merged**